SirMangler/pcsx2
# WINRT/UWP: Code Cleanups & Patches Tweaks

### Description of Changes
- Cleans up a bit of code in the WINRT `App.cpp` and `pcsx2-winrt.vcxproj` files, while also making some tweaks to the `common.props` file to smooth over the UWP build process a bit.
- Debloats the `PCSX2_qt.sln` file significantly, removing unnecessary lines regarding the unsupported CPU architectures. Instead of telling Visual Studio to use x64 even when ARM/ARM64/x86 are selected, we can just remove them so only x64 can be used. Same result but with less lines of code needed.
- Fixes an inconsistent line in the `GSDevice11.cpp` file to maintain parity with upstream.
- Adds back the 60FPS and DNAS Patches settings so these patches are still working with the latest version.
- Adds a new `patches.zip` file that includes the 4 types of patches (widescreen, no-interlacing, 60 FPS, and DNAS) at once, while also being up to date with the latest [patches files](https://github.com/PCSX2/pcsx2_patches). Also makes it possible to make a build without providing the file yourself. I also added the Python script I used to create the patches included in the zip file.

### Rationale behind Changes
- Cleaner code.
- Adds back removed features.
- Streamlines the build process.

### Suggested Testing Steps
- Make sure the app still builds just fine.
- Make sure the patches work as they should.
